### DISCUSSION TITLE
Can you share the source code for the globe on your landing page?

### DISCUSSION THREAD

**Discussion by j16h** [vercel/community#2714] [Help] [answered]
### Summary

On your landing page www.vercel.com you have that beautiful animated globe. Could you share the source code for it, please 🥺🙏

### Example

www.vercel.com 

### Steps to Reproduce

scroll to the bottom of your page :)

**Comment by leerob** [comment, marked as answer]
- https://github.com/shuding/cobe
- https://cobe.vercel.app/
